chart/templates/_helpers.tpl

```diff
@@ -3,5 +3,9 @@
 {{- end -}}
 
 {{- define "nexus-iq-server-ha.iqServerImage" -}}
-{{- if .Values.iq_server.imageRegistry }}{{ .Values.iq_server.imageRegistry }}/{{ .Values.iq_server.image }}:{{ .Values.iq_server.tag }}{{- else }}{{ .Values.iq_server.image }}:{{ .Values.iq_server.tag }}{{- end }}
+{{- if (.Values.iq_server).imageRegistry }}{{ (.Values.iq_server).imageRegistry }}/{{ (.Values.iq_server).image }}:{{ (.Values.iq_server).tag }}{{- else }}{{ (.Values.iq_server).image }}:{{ (.Values.iq_server).tag }}{{- end }}
+{{- end -}}
+
+{{- define "nexus-iq-server-ha.busyboxImage" -}}
+{{- if (.Values.busybox).imageRegistry }}{{ (.Values.busybox).imageRegistry }}/{{ (.Values.busybox).image }}:{{ (.Values.busybox).tag }}{{- else }}{{ (.Values.busybox).image }}:{{ (.Values.busybox).tag }}{{- end }}
```

Comment:
Couldn't this be simplified by just wrapping the registry part and the slash in the conditional?

Reply:
> Couldn't this be simplified by just wrapping the registry part and the slash in the conditional?

I think moving the slash inside would require some concatenation function like `printf`, I could be misunderstanding though